lib/kino/js/live/server.ex

```diff
@@ -23,6 +23,19 @@ defmodule Kino.JS.Live.Server do
     :ok
   end
 
+  def send_event(ctx, origin, event, payload) do
+    ref = ctx.__private__.ref
+
+    pid =
+      case ctx.__private__.clients[origin] do
+        {pid, _} -> pid
+        _ -> raise "could not find a connected client with origin #{inspect(origin)}"
```

Comment:
I think it may be better to just execute a no-op because you can always have race conditions: i.e. between you computing the message to send and sending it, the client may have been disconnected.

That's why sending processes in Erlang almost always no-op if the process is dead.

Reply:
Good call!